### DISCUSSION TITLE
Add Option to trim Multiple Glyph Block at once....

### DISCUSSION THREAD

**Discussion by hello-jai** [Krishnagopal-Sinha/better-nothing-glyph-composer#27] [Ideas]
Hey try to add this in next version.... Multi-Select Option should also be able to trim multiple glyph block at once

**Comment by Krishnagopal-Sinha** [comment]
> Hey try to add this in next version.... Multi-Select Option should also be able to trim multiple glyph block at once

Hi, thank you for this feature request, this has been added v1.1.3 onwards, and this change is live. Feel free to check it out :D
